src/js/event-target.js

```diff
@@ -11,7 +11,7 @@ EventTarget.prototype.on = function(type, fn) {
   // Remove the addEventListener alias before calling Events.on
   // so we don't get into an infinite type loop
   let ael = this.addEventListener;
-  this.addEventListener = Function.prototype;
+  this.addEventListener = () => {};
```

Comment:
Point for discussion: should we be careful about using fat-arrows all over the place? I know calling addEventListener() with a non-`this` receiver is a weird thing to do but we could use a plain function here and avoid possibly subverting people's expectations.